flow/embedded_views.h

```diff
@@ -56,7 +56,7 @@ class Mutator {
         alpha_ = other.alpha_;
         break;
       case kBackdropFilter:
-        filter_ = other.filter_;
+        filter_ = other.filter_->shared();
```

Comment:
For the copy constructor you can just copy the shared_ptr without having to recreate it with the shared() method.

Reply:
What do you mean by copy the shared_ptr?

Reply:
`filter_ = other.filter_`

Why do you call shared() there? You already have a shared_ptr.

Reply:
Does `filter_ = other.filter_` copy the raw pointer inside the shared_ptr? The idea of this copy constructor is that the filters are different objects, changing the value of the original filter will not affect the value of the copied filter.

Reply:
No deep copy. It points to a new filter. It no longer points to the old filter.